diff --git a/node_modules/two.js/types.d.ts b/node_modules/two.js/types.d.ts
index daeeda0..1a0dd76 100644
--- a/node_modules/two.js/types.d.ts
+++ b/node_modules/two.js/types.d.ts
@@ -2892,7 +2892,7 @@ declare module 'two.js/src/path' {
      * @description A list of numbers that represent the repeated dash length and dash space applied to the stroke of the text.
      * @see {@link https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray} for more information on the SVG stroke-dasharray attribute.
      */
-    dashes: { [key: number]: number; offset: number };
+    dashes: Array<number> // Patched for Spherical Easel
     /**
      * @name Two.Path#copy
      * @function
